articles/how-to-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/how-to-build-astatic-site-with-gatsbyjs/
+title: How to build a static site with GatsbyJs
+description:
+author: jethro-magaji
+date: 2020-10-19T00:00:00-08:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-to-build-astatic-site-with-gatsbyjs/hero.png
+    alt: hero example image computer Node.js
+---
+When I first heard about GatsbyJs, I was confused about what it was, and how I would get started. As time went on, I learned more about it and found out it is awesome. My goal for this article is not to confuse you about GatsbyJs, but to help you get started with building your very first static site using GatsbyJs.
+<!--more-->
+A paradigm shift has begun in the web development world, where static sites are now becoming more of a trend over dynamic sites. Building a dynamic site that required you to set it up and/or maintaining it, is now obsolete.
+
+This is not an issue when building static sites. Due to cloud service companies like [Amazon](https://aws.amazon.com/), [Google](https://cloud.google.com/), [Netlify](https://www.netlify.com/), [Heruko](https://www.heroku.com/), and [Gaysby Cloud](https://www.gatsbyjs.com/cloud/) etc. Having worked on making software development easier, faster, and more scalable for developers.
+
+WHen compared to static sites, dynamic sites are slow to load and can cause high bounce rate on a page by users, now that’s where GatsbyJs comes into play; it is a powerful and easy static site generator.
```

Comment:
When*

You should give examples (statistics). Compare the speed between a static and dynamic site.

Reply:
> When*
> 
> You should give examples (statistics). Compare the speed between a static and dynamic site.

@Jethro-magaji 

Just wanted to make sure we are implementing this suggestion as well 🚀

Reply:
Resolved

Reply:
I don't see any statistics comparing the speed between static and dynamic sites, @Jethro-magaji. Have you commited and pushed all your changes?

Reply:
I have removed the paragraph.